oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -531,18 +531,134 @@ int DumpHandler::read(uint32_t offset, uint32_t& length, Response& response,
     return readFile(resDumpDirPath, offset, length, response);
 }
 
-int DumpHandler::fileAckWithMetaData(uint32_t metaDataValue1,
+int DumpHandler::fileAckWithMetaData(uint8_t /*fileStatus*/,
+                                     uint32_t metaDataValue1,
                                      uint32_t metaDataValue2,
                                      uint32_t /*metaDataValue3*/,
                                      uint32_t /*metaDataValue4*/)
 {
     auto path = findDumpObjPath(fileHandle);
-    uint8_t fileStatus = (uint8_t)metaDataValue2;
+    uint8_t statusCode = (uint8_t)metaDataValue2;
     if (dumpType == PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS)
     {
+        DBusMapping dbusMapping;
+        dbusMapping.objectPath = "/xyz/openbmc_project/dump/resource/entry/" +
+                                 std::to_string(fileHandle);
+        dbusMapping.interface = "com.ibm.Dump.Entry.Resource";
+        dbusMapping.propertyName = "DumpRequestStatus";
+        dbusMapping.propertyType = "string";
+
         std::cout << "fileAckWithMetaData with token: " << metaDataValue1
                   << " and status: " << metaDataValue2 << std::endl;
-        if (fileStatus != PLDM_SUCCESS)
+        if (statusCode == DumpRequestStatus::ResourceSelectorInvalid)
```

Comment:
Change this code like
if (statusCode == DumpRequestStatus::ResourceSelectorInvalid)
{
            pldm::utils::PropertyValue value =
                "com.ibm.Dump.Entry.Resource.HostResponse.ResourceSelectorInvalid";
}
else if()
{
}
...
Then do a a single call to set property